justfile

```diff
@@ -2,6 +2,14 @@
 default:
   @just --list --unsorted
 
+lint:
+	@just clippy bolt-cli
+	@just clippy bolt-boost
+	@just clippy bolt-sidecar
+
+clippy package:
+	cd {{ package }} && cargo clippy --all-targets --all-features -- -D warnings
```

Comment:
can you add short doc comments starting with `# ` on the line above each command? it's useful when seeing the
list of available commands with `just`